airbyte-integrations/connectors/source-hubspot/Dockerfile

```diff
@@ -34,5 +34,5 @@ COPY source_hubspot ./source_hubspot
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=1.5.1
+LABEL io.airbyte.version=1.5.2
```

Comment:
```suggestion
LABEL io.airbyte.version=1.6.0
```

Reply:
https://docs.airbyte.com/contributing-to-airbyte/resources/pull-requests-handbook/#examples